package.json

```diff
@@ -67,6 +67,17 @@
   "standard": {
     "parser": "babel-eslint"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
+    }
+  },
+  "lint-staged": {
+    "{src,__tests__,__fixtures__,config}/**/*.js": [
+      "eslint --ext .js --ext .jsx"
```

Comment:
Rather than just running eslint, we could also add all lint fixes automatically to the commit:
https://github.com/okonet/lint-staged#automatically-fix-code-style-with---fix-and-add-to-commit